pegasus/sites.v3/code.org/public/donate/index-draft.haml

```diff
@@ -0,0 +1,102 @@
+---
+title: Donate
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+-# TODO - Update this to donate-styles.css
```

Comment:
Is this for after donate-styles gets deprecated?

Reply:
Yep! I'm going to rename `donate-styles-draft` to `donate-styles` when the time comes.